code/modules/xenomorph/xeno_structures.dm

```diff
@@ -397,26 +397,41 @@ TUNNEL
 
 ///Here we pick a tunnel to go to, then travel to that tunnel and peep out, confirming whether or not we want to emerge or go to another tunnel.
 /obj/structure/xeno/tunnel/proc/pick_a_tunnel(mob/living/carbon/xenomorph/M)
-	var/obj/structure/xeno/tunnel/targettunnel = tgui_input_list(M, "Choose a tunnel to crawl to", "Tunnel", GLOB.xeno_tunnels_by_hive[hivenumber])
+	to_chat(M, span_notice("Select a tunnel to go to."))
```

Comment:
Is there a way to cancel out once you've opened the map, or is it just click somewhere with no tunnels to not go anywhere?

Reply:
no

Reply:
You can click the edge of the map to exit tho yea